argocd_application_sets/managing-app-sets-in-non-control-plane-namespaces.adoc

```diff
@@ -32,7 +32,7 @@ include::modules/gitops-enabling-the-application-set-resources-in-non-control-pl
 include::modules/gitops-allowing-scm-providers.adoc[leveloffset=+1]
 
 [role="_additional-resources"]
-[id="additional-resources_enabling-app-sets-in-any-non-control-plane-namespaces"]
+[id="additional-resources_managing-app-sets-in-non-control-plane-namespaces_{context}"]
```

Comment:
We have mentioned the exact content in id and then, we are agin using the context variable. Why?

See here: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies 

Anchor id should be in this format:
[id="additional-resources_{context}"]
== Additional resources